test/html/cases/alias.ml

```diff
@@ -0,0 +1,7 @@
+module Foo__X = struct
+  (** Module Foo__X documentation. This should appear in the documentation
+      for the alias to this module 'X' *)
```

Comment:
That's incorrect, that docstring is actually attached to the type `t`, not to the module. You want to add a blank line.